nx.json

```diff
@@ -37,7 +43,7 @@
       "dependsOn": [ "^package" ]
     },
     "test": {
-      "inputs": [ "default", "^default" ],
+      "inputs": [ "default", "^production" ],
```

Comment:
including ^default (i.e. test + prod codebase of dependencies) was resulting in very long spin in local `yarn test`.
When tests executed they were writing files to each project directory, that caused cache invalidation and nx was trying to rebuild and re test them.